packages/api-server/db_migrations/004_logging.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TYPE user_action_enum AS ENUM ('add', 'update', 'delete');
+
+CREATE TABLE IF NOT EXISTS user_logs (
+  user_id INT NOT NULL,
+  server_id INT NOT NULL,
+  user_action user_action_enum NOT NULL,
+  details VARCHAR (1024),
+  created_on TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
+);
+
+ALTER TABLE portals ADD COLUMN updated_by INT;
+ALTER TABLE portals ADD COLUMN updated_ON TIMESTAMP WITH TIME ZONE;
```

Comment:
Can't these two be fetched from the new user_logs table?

Reply:
Yeah. I think I just added these out of habit. I don't even remember updating them.